library/common-library/templates/_verbose-log.tpl

```diff
@@ -0,0 +1,26 @@
+{{- /*
+Abstraction of the verbose toggle.
+This helper allows to override the global `.global.verboseLog` with the value of `.verboseLog`.
+Returns "true" if `verbose` is enabled, otherwise "" (empty string)
+*/ -}}
+{{- define "common.verboseLog" -}}
+{{- /* `get` will return "" (empty string) if value is not found, and the value otherwise, so we can type-assert with kindIs */ -}}
+{{- if (get .Values "verboseLog" | kindIs "bool") -}}
+    {{- if .Values.verboseLog -}}
+        {{- /*
+            We want only to return when this is true, returning `false` here will template "false" (string) when doing
+            an `(include "common.verboseLog" .)`, which is not an "empty string" so it is `true` if it is used
+            as an evaluation somewhere else.
+        */ -}}
+        {{- .Values.verboseLog -}}
+    {{- end -}}
```

Comment:
In other PRs, we print the `false` string rather than returning empty. This does not allow to `if` the result of this helper, but does allow to render it unconditionally which is allegedly a more common use case.

Reply:
We are using `with` in the templates so it is perfect to print `true` or `""`. In all the cases we have faced until now the default is `false`. See `hostNetwork` [here](https://github.com/newrelic/helm-charts/blob/master/library/CHART-TEMPLATE/tests/common_library_hostnetwork_test.yaml), `nrStaging` [here](https://github.com/newrelic/helm-charts/blob/master/library/common-library/templates/_staging.tpl) or this helper for adding a verbose flag. In case of them, you need an explicit `false` you can use `default` as the templates do [here](https://github.com/newrelic/helm-charts/pull/752/files#diff-c4d3c722fb93e10d6c397bc0513bbff9d34b0b4337f30130c41c49f6eea99566R11).

This comment has raised the issue that `hostNetwork` is printing "false" and that `staging` has no tests but other toggles we have has exactly the same behaviour (and I know becuase I copy&pasted them myself 😅)